test/BUILD

```diff
@@ -58,6 +56,33 @@ jasmine_node_test(
     data = ["//:test_files"],
 )
 
+test_suite(
+    name = "small_tests",
+    tests = [
+        ":e2e_test",
+        ":unit_test",
+    ],
+)
+
+ts_library(
+    name = "closure_e2e_test_lib",
+    srcs = [
+        "closure.ts",
+        "e2e_test.ts",
+    ],
+    tsconfig = "//:tsconfig.json",
+    deps = [
+        ":test_support",
+        "//src",
+    ],
+)
+
+jasmine_node_test(
+    name = "closure_e2e_test",
```

Comment:
You'll need to duplicate this in the internal test/BUILD file when you import it.

Reply:
Ack.